### ISSUE TITLE
Trouble resuming a download

### ISSUE THREAD

**Issue by pfrazee** [joehand/dat-next#66]
Here's the flow I went through:

```
./bin/cli.js clone 04ed0b08ff595a992a594ad1ab624072646467ec7eda2dc40e4aa512e49cb196 ~/tmp/substack
```

Downloaded all metadata and a little content, then closed. Tried to resume with the same command and got:

```
Archive exists in directory, cannot overwrite
```

So then I tried

```
./bin/cli.js pull --dir=~/tmp/substack
```

and that gave me:

```
OpenError: IO error: ~/tmp/substack/.dat/LOCK: No such file or directory
    at /Users/paulfrazee/dat-next/node_modules/levelup/lib/levelup.js:119:34
    at /Users/paulfrazee/dat-next/node_modules/leveldown/node_modules/abstract-leveldown/abstract-leveldown.js:39:16
```

I got the same error with `sync`

**Comment by pfrazee** [comment]
No rush on solving, just wanted to report

**Comment by joehand** [comment]
Should be fixed with `dat-node v1.1.1`
